server/quota/quota.go

```diff
@@ -17,35 +17,32 @@ package quota
 import (
 	"context"
 	"sync"
-	"time"
 
 	"github.com/rs/zerolog/log"
 	"github.com/tigrisdata/tigris/errors"
 
-	"github.com/tigrisdata/tigris/server/request"
-
-	"github.com/tigrisdata/tigris/schema"
-
-	ulog "github.com/tigrisdata/tigris/util/log"
-
 	"github.com/tigrisdata/tigris/server/config"
 	"github.com/tigrisdata/tigris/server/metadata"
-	"github.com/tigrisdata/tigris/server/metrics"
-	"github.com/tigrisdata/tigris/server/transaction"
 	"go.uber.org/atomic"
-	"golang.org/x/time/rate"
 )
 
 var (
-	ErrRateExceeded        = errors.ResourceExhausted("request rate limit exceeded")
-	ErrThroughputExceeded  = errors.ResourceExhausted("request throughput limit exceeded")
-	ErrStorageSizeExceeded = errors.ResourceExhausted("data size limit exceeded")
+	ErrReadRateExceeded        = errors.ResourceExhausted("request read rate limit exceeded")
```

Comment:
why not just use `api.Errorf(api.Code_ResourceExhausted...)`?

Reply:
`errors.ResourceExhausted(` is wrapper of `api.Errorf(api.Code_ResourceExhausted...`